src/api/utils/summary_cache.py

```diff
@@ -170,31 +298,13 @@ def _update_bleaching_qc_summary(
     _update_records(bleaching_se, BleachingQCSEModel, created_on, suggested_citation, skip_updates)
 
 
-def _update_project_summary_sample_event(project_id, skip_test_project=True):
```

Comment:
Seems like we should be consistent. This function gets deleted (and the sql equiv added), whereas in other cases the corresponding non-sql function isn't deleted. Should be one or the other.